x-pack/plugins/apm/server/lib/errors/distribution/get_buckets.ts

```diff
@@ -15,12 +19,13 @@ export async function getBuckets({
   setup
 }: {
   serviceName: string;
-  groupId: string;
+  groupId?: string;
```

Comment:
`groupId` should also be optional in https://github.com/elastic/kibana/blob/31cd2bc370f5896325334eb99e28e63e6e35a3da/x-pack/plugins/apm/server/lib/errors/distribution/get_distribution.ts#L30